### ISSUE TITLE
Bug in jscodemod: maxTokens is negative

### ISSUE THREAD

**Issue by vicero** [NickHeiner/jscodemod#45]
First I'd like to say thanks for this tool, it looks incredible!

I can't seem to get the tool to run, however.  I created an OpenAI API token and ran jscodemod on a subfolder of my project:

```
OPENAI_ORG_ID=org-xxx OPENAI_API_KEY=sk-xxx jscodemod --prompt "Translate the file above from ES5 to modern JS" src/admin/webhooks/**/*.js
AssertionError [ERR_ASSERTION]: Bug in jscodemod: maxTokens is negative
    at OpenAIBatchProcessor.handleRequestReady (/home/james/.nvm/versions/node/v16.18.0/lib/node_modules/@nick.heiner/jscodemod/build/run-ai-codemod.js:291:30)
    at OpenAIAPIRateLimiter.innerAttemptCall (/home/james/.nvm/versions/node/v16.18.0/lib/node_modules/@nick.heiner/jscodemod/build/run-ai-codemod.js:99:34)
    at Timeout._onTimeout (/home/james/.nvm/versions/node/v16.18.0/lib/node_modules/@nick.heiner/jscodemod/node_modules/p-debounce/index.js:21:56)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
```

**Comment by NickHeiner** [comment]
Heh, I'm glad I labelled it as a bug in that error message, because I believe that's what this is. 

Are you able to share the file you're running on that triggers this input? I see you're using a glob pattern; are you able to tell from the logs which file in particular is problematic? You could try running on a smaller glob pattern.

You can also run with env var `loglevel=trace` or `loglevel=debug` to get more info.

**Comment by NickHeiner** [comment]
I think this will be fixed by #46.
